gen/definitions/profile_parcels/transport_routing_bgp.yaml

```diff
@@ -0,0 +1,334 @@
+---
+name: Transport Routing BGP
+rest_endpoint: /v1/feature-profile/sdwan/transport/%v/routing/bgp
+minimum_version: 20.12.0
+test_tags: [SDWAN_2012]
+attributes:
+  - tf_name: feature_profile_id
+    reference: true
+    type: String
+    description: Feature Profile ID
+    example: f6dd22c8-0b4f-496c-9a0b-6813d1f8b8ac
+    test_value: sdwan_transport_feature_profile.test.id
+  - model_name: asNum
+    tf_name: as_number
+    mandatory: true
+    example: 429
+  - model_name: routerId
+    tf_name: router_id
+    example: "1.2.3.4"
+  - model_name: propagateAspath
+    tf_name: propagate_as_path
+    example: false
+  - model_name: propagateCommunity
+    example: false
+  - model_name: external
+    tf_name: external_routes_distance
+    example: 20
+  - model_name: internal
+    tf_name: internal_routes_distance
+    example: 200
+  - model_name: local
+    tf_name: local_routes_distance
+    example: 20
+  - model_name: keepalive
+    tf_name: keepalive_time
+    example: 60
+  - model_name: holdtime
+    tf_name: hold_time
+    example: 180
+  - model_name: alwaysCompare
+    tf_name: compare_med
```

Comment:
From what I remember, the "always" is important here --> `tf_name: always_compare_med`